cpp/modmesh/buffer/pymod/wrap_SimpleArray.cpp

```diff
@@ -87,6 +87,8 @@ class MODMESH_PYTHON_WRAPPER_VISIBILITY WrapSimpleArray
                     }),
                 py::arg("array"))
             .def_buffer(&property_helper::get_buffer_info)
+            .def("clone", [](wrapped_type const & self)
+                 { return wrapped_type(self); }) // cloning the object using the copy constructor. never add the clone method to the C++ class.
```

Comment:
When the second argument is a lambda, please move it to the next line, so that the indentation of the lambda body will be aligned with the character the lambda starts, like:

```cpp
            .def("clone",
                 [](wrapped_type const & self)
                 { return wrapped_type(self); }) // cloning the object using the copy constructor. never add the clone method to the C++ class.
```